protocol/avdl/chat1/local.avdl

```diff
@@ -409,7 +448,7 @@ protocol local {
 
   // Get messages by ID.
   @lint("ignore")
-  GetMessagesLocalRes GetMessagesLocal(ConversationID conversationID, array<MessageID> messageIDs, keybase1.TLFIdentifyBehavior identifyBehavior);
+  GetMessagesLocalRes GetMessagesLocal(ConversationID conversationID, array<MessageID> messageIDs, keybase1.TLFIdentifyBehavior identifyBehavior, boolean disableResolveSupersedes);
```

Comment:
Can we keep the `identifyBehavior` last here? It is just the convention everywhere (no great reason), but it woul dbe nice to keep that consistent.